client/src/App.scss

```diff
@@ -1,6 +1,48 @@
 @import '~antd/dist/antd.css';
 @import './styles/colors.scss';
 
+:root {
+  [data-color-theme='green'] {
+    --color-primary: #00b295;
```

Comment:
I couldn't get these to work with SCSS variables for some reason, might be worth looking into?